config/checkstyle_checks.xml

```diff
@@ -565,6 +570,11 @@
     <module name="MissingJavadocMethod">
       <property name="allowMissingPropertyJavadoc" value="true"/>
     </module>
+    <module name="MissingJavadocMethod">
+      <property name="id" value="missingPrivateMethodsJavadoc"/>
+      <property name="allowMissingPropertyJavadoc" value="true"/>
+      <property name="scope" value="private"/>
```

Comment:
Same.

Reply:
Here we need a second instance since check can accept only one scope. Default is public, so one more instance of check is added with same properties but different scope

Reply:
`private` should already include `public`. Only 1 scope should be needed in this case.
https://checkstyle.org/property_types.html#Scope

> Using a specific scope means not only will that modifier be examined, but also all the modifiers listed above it in the previous list. Specifying public means only items with public modifiers are checked. Specifying protected means only public and protected modifiers are checked.

Reply:
I didn't know that. I have made changes